crates/pop-chains/src/new_chain.rs

```diff
@@ -73,9 +73,6 @@ pub fn instantiate_standard_template(
 		&target.join("node/src/chain_spec.rs"),
 		chainspec.render().expect("infallible").as_ref(),
 	)?;
-	// Add network configuration
```

Comment:
For devs using the templates in a version 2506 or below this, this is a breaking change that it won't create a network.toml
Because we have the version in https://github.com/r0gue-io/pop-cli/blob/main/crates/pop-chains/src/new_chain.rs#L46 we can check the version and generate the network.toml if is an older version